core/src/services/s3/core.rs

```diff
@@ -664,7 +682,7 @@ impl S3Core {
 
     pub async fn s3_delete_objects(
         &self,
-        paths: Vec<String>,
+        paths: Vec<(String, Option<String>)>,
```

Comment:
We don't need to support delete multiple versions for now. This should be considered in deleter.

Reply:
Deleter hasn't been implemented yet, so I changed the batch.
It's ok to keep it as is.

Reply:
> It's ok to keep it as is.

Let's keep batch api as is.